tests/unit/challenges/test_utils.py

```diff
@@ -1,18 +1,180 @@
+import boto3
+import json
+import logging
 import os
-import unittest
+import mock
```

Comment:
@abinav62 can you reorder imports on lines 4 & 5 lexicographically.

Reply:
@Ram81 I have done it.